apps/web/src/routes/dashboard/_dashboard/-components/menu-desktop.tsx

```diff
@@ -60,6 +60,7 @@ export const MenuDesktop = ({
   const coursesPath = '/dashboard/courses';
   const profilePath = '/dashboard/profile';
   const administrationPath = '/dashboard/administration';
+  const administrationBookingsPath = '/dashboard/a-bookings';
```

Comment:
We will need to change it -> once my PR #849 is merged, check how it is done for admin routes